### PR TITLE
IRIS-4788 | Make sure blockquotes and headings look nice on dark theme

### PR DESCRIPTION
**Pull request by slayful**
## Links

* http://wikia-inc.atlassian.net/browse/IRIS-4788

### CODE REVIEW

File: front/main/app/styles/runtime.scss [hunk 1]
```diff
@@ -20,3 +20,6 @@
 @import 'auth/app/styles/base/input';
 @import 'auth/app/styles/module/forms';
 @import 'auth/app/styles/module/form-icons';
+
+// discussions specifics
+@import '_discussion-quotes';
```
Comment (rogatty) [inline front/main/app/styles/runtime.scss]: We do `@import 'component/index';` above so this isn't needed.

File: front/main/app/styles/theme/_dark.scss [hunk 2]
```diff
@@ -172,6 +171,10 @@ body.dark-theme {
 						color: $dark-discussion-post-title-color;
 					}
 				}
+
+				h1,h2,h3,h4,h5,h6 {
```
Comment (rogatty) [inline front/main/app/styles/theme/_dark.scss]: Where do these tags come from? Can we use put classes in them and use here?

File: front/main/app/styles/_discussion-quotes.scss [hunk 3]
```diff
@@ -12,3 +12,11 @@
 	-webkit-margin-start: 0;
 	-webkit-margin-end: 0;
 }
+
+.discussions blockquote {
+	border-left: 4px solid $color-links;
+	color: #999;
+	p {
+		color: #999;
```
Comment (rogatty) [inline front/main/app/styles/_discussion-quotes.scss]: I think 3 times is enough for a variable.

File: front/main/app/styles/theme/_dark.scss (hunk 2 above)
Comment (slayful) [inline front/main/app/styles/theme/_dark.scss]: We cannot as these come from migrated forums.

File: front/main/app/styles/component/discussion/_discussion-quotes.scss [hunk 4]
```diff
@@ -12,3 +12,11 @@
 	-webkit-margin-start: 0;
 	-webkit-margin-end: 0;
 }
+
+.discussions blockquote {
+	border-left: 4px solid $color-links;
+	color: #999;
+	p {
+		color: #999;
```
Comment (rogatty) [inline front/main/app/styles/component/discussion/_discussion-quotes.scss]: I think 3 times is enough for a variable.

File: front/main/app/styles/theme/_dark.scss (hunk 2 above)
Comment (rogatty) [inline front/main/app/styles/theme/_dark.scss]: How about an explanatory comment, then?

**Pull request merged**